content/applications/inventory_and_mrp/manufacturing/subcontracting/dropship_subcontracting_lead_times.rst

```diff
@@ -0,0 +1,187 @@
+   should not be viewed as guarantees.
+
+Configuration
+=============
+
+When using the :doc:`Dropship Subcontractor on Order <subcontracting_dropship>` route, a company is
+responsible for purchasing the necessary components from a vendor and having them dropshipped
+directly to the subcontractor.
+
+This means that, in addition to the amount of time it takes the subcontractor to manufacture and
+deliver the product, it is also necessary to consider how long it takes the component vendor to
+dropship the components to the subcontractor.
+
+By assigning a product's subcontractor a *delivery lead time*, specifying a *manufacturing lead
+time* on the product's BoM, and assigning the vendor of the components an additional *delivery lead
+time*, *Dropship Subcontractor* orders for the product's components display the deadline for
+confirming a dropship order to send the components to the subcontractor.
+
+Product delivery lead time
+--------------------------
+
+To set a delivery lead time for a product's subcontractor, navigate to :guilabel:`Inventory app -->
+Products --> Products`, and select a subcontracted product.
+
+Then, select the :guilabel:`Purchase` tab on the product's page. If the subcontractor has not
+already been added as a vendor, do so now by clicking :guilabel:`Add a line`, and selecting the
+subcontractor in the :guilabel:`Vendor` column.
+
+Once the subcontractor has been added, enter the number of days it takes them to manufacture and
+deliver the product, in the :guilabel:`Delivery Lead Time` column.
+
+.. image:: dropship_subcontracting_lead_times/delivery-lead-time.png
+   :align: center
+   :alt: The Delivery Lead Time field for a subcontractor, on the Purchase tab of a product page.
+
+Product manufacturing lead time
+-------------------------------
+
+Next, navigate to the product's |BoM| by clicking the :guilabel:`Bill of Materials` smart button at
+the top of the product's page. Then, select a BoM from the list.
+
+On the |BoM|, select the :guilabel:`Miscellaneous` tab. In the :guilabel:`Manuf. Lead Time` field,
+enter the same number of days that was entered in the :guilabel:`Delivery Lead Time` field of the
+|BoM|'s product.
+
+.. image:: dropship_subcontracting_lead_times/manufacturing-lead-time.png
+   :align: center
+   :alt: The Manuf. Lead Time field on a product's BoM.
+
+While not all of these days will actually be used for manufacturing by the subcontractor, setting
+the same number of days in each field tells Odoo that the subcontractor must receive the components
+and begin production by the start of the product's delivery lead time. This gives the subcontractor
+enough time to both manufacture and deliver the product.
+
+Component delivery lead time
+----------------------------
+
+From the product's BoM, navigate to each component by clicking on the component in the
+:guilabel:`Components` tab, and then clicking the :guilabel:`➡️ (Internal link)` button to the right
```

Comment:
I think @samueljlieber can give the best advice on this, but I think we're moving away from using emojis. Instead, we're hover over the icon, right clicking, and selecting "Inspect" to find the icon name. This one though, I wasn't able to get working.

![image](https://github.com/odoo/documentation/assets/23611711/21138391-af2c-468d-abe3-9945048ca8b2)

It looks like the icon code is "oi-arrow-right", but I couldn't get it working. This is what I tried:
```suggestion
:guilabel:`Components` tab, and then clicking the :icon:`oi-arrow-right` :guilabel:`(Internal link)` button to the right
```